cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -1,19 +1,16 @@
 import * as c from '../../support/constants.js'
 import {ResearcherPermissionFormsElements as pf} from "../../support/elements/researcher_permission_forms_elements";
 
-const TEST_PROJECT_NAME = 'Test Project',
+const TEST_PROJECT_NAME = 'Cypress Test Project',
+    OTHER_TEST_PROJECT_NAME = 'Test Project',
     TEST_PERMISSION_FORM_NAME = 'PermForm4CypressTests',
-    TEST_PERMISSION_FORM_URL  = 'concord.org'
-    // TODO: I don't know the password for c.RESEARCHER_USERNAME (i.e. 'researcher'), so I've created my own, for now
```

Comment:
Maybe Scott or Saravanan knows the login credentials.

Reply:
Saravanan did know them - that's why I deleted those 2 lines.